tools/tests/test_lint.py

```
@@ -188,20 +188,64 @@ def test_ledger_row_missing_field_is_a_finding(tmp_path):
     assert len(findings) == 1 and "ledger" in findings[0] and "missing field" in findings[0]
 
 
-def test_valid_ledger_row_is_clean(tmp_path):
+def _ledger_row(**overrides):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_ledger_row`

(ANN202)